e2e/utils/flows/deleteDashboard.ts

```diff
@@ -29,6 +29,14 @@ export const deleteDashboard = ({ quick = false, title, uid }: DeleteDashboardCo
 
 const quickDelete = (uid: string) => {
   cy.request('DELETE', fromBaseUrl(`/api/dashboards/uid/${uid}`));
+  cy.window().then((win: Cypress.AUTWindow) => {
+    if (
+      win.grafanaBootData.settings.featureToggles.dashboardRestore &&
+      win.grafanaBootData.settings.featureToggles.dashboardRestoreUI
+    ) {
+      cy.request('DELETE', fromBaseUrl(`/api/dashboards/uid/${uid}/trash`));
```

Comment:
Does this work for non-trashed dashboards?

Reply:
You mean the endpoint? I don't believe so, but we have soft deleted this dashboard in the endpoint above